plugins/rocks.xml.webhelp/xsl/dita2xmlrocksCoverImpl_template.xsl

```diff
@@ -3,7 +3,7 @@
 
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/cover.xsl"/>
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_common.xsl"/>
-  <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_tables.xsl"/>
+  <xsl:import href="plugin:rocks.xml.webhelp:xsl/tables.xsl"/>
```

Comment:
I don't think we need tables transformation imported from the cover xsl.

Reply:
done